src/com/model/BanknotesCounter.java

```diff
@@ -0,0 +1,31 @@
+package com.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class BanknotesCounter {
+    // Allowed banknotes to be deposited;
+    public final int[] ALLOWED_BANKNOTES = {1, 3, 5, 10, 20, 100};
+    // The key is the denomination of the banknote. Value - its quantity in stock;
+    public Map<Integer, Integer> banknotesMap = new HashMap<>();
```

Comment:
Please avoid using public class fields -- in Java, this is a rare approach and is most of the time considered a bad practice. Always use *getter* and *setter* methods, in IntelliJ IDEA, you just press Alt+Insert on your keyboard (in Windows) and select "Getter and Setter" from the context menu to create those semi-automatically